Added UI

justakit opened the pull request:
Added basic UI functionality

justakit committed:
Added UI

justakit committed:
Deleted some Prints

justakit committed:
Rename variables and functions to normal names

qmlReg => qmlRegister
sendRegD => checkRegisterData
sendRegIsValid => registerDataIsValid
initQmlReg => initQmlRegister

qmlLog => qmlLogin
sendLogD => checkLoginData
sendLogIsValid => loginDataIsValid
initQmlLog => initQmlLogin

8tomat8 commented on client/ui/uimain.go:
```diff
@@ -0,0 +1,34 @@
+
+func CreateWindow() {
+	widgets.NewQApplication(len(os.Args), os.Args)
+	var layout = widgets.NewQHBoxLayout()
```
You can use short syntax for simplicity.
For example:
`layout := widgets.NewQHBoxLayout()`

justakit replied:
If it possible to leave "**var** varName" for better highlighting of variables?

8tomat8 commented on client/main.go:
```diff
@@ -1,7 +1,9 @@
 
-import "fmt"
+import (
+	"SSU-Golang-252-Chat/client/ui"
```
Import path should look like this `"github.com/8tomat8/SSU-Golang-252-Chat/client/ui"`

justakit replied:
Import path changed

8tomat8 reviewed (approved):
Looks good to me. Please look for some comments

justakit committed:
Change import path "SSU-Golang-252-Chat/client/ui" to "github.com/8tomat8/SSU-Golang-252-Chat/client/ui"

Added slot "addContact" to contacts.go

Added instruction for installation therecipe/qt library

justakit committed:
Merge branch 'master' into UIDevelopment
